app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -138,8 +143,13 @@ class BookmarksTest {
             verifyBookmarkNameEditBox()
             verifyBookmarkURLEditBox()
             verifyParentFolderSelector()
-            navigateUp()
-            verifyBookmarksMenuView()
+            changeBookmarkTitle(bookmarkTitle)
+            changeBookmarkUrl(bookmarkUrl)
+            saveEditBookmark()
+            getInstrumentation().waitForIdleSync()
```

Comment:
Similar to this wait, are we waiting for the RecyclerView to have an item count? If so you can try using https://github.com/mozilla-mobile/fenix/blob/8eb1520786f4c94eae7c0fe9498ae61a47d8c628/app/src/androidTest/java/org/mozilla/fenix/helpers/RecyclerViewIdlingResource.kt

Reply:
done. Please review.